Cargo.toml

```diff
@@ -15,3 +15,7 @@ exclude = [".gitignore"]
 
 [dev-dependencies]
 clap = { version = "4.5.3", features = ["derive"] }
+
+[dependencies]
+once_cell = "1.19.0"
```

Comment:
[Q] 
1. Are using external crates allowed?
2. If using crates is possible, is it better to keep them in `dev-dependencies` instead?